websites/M/Mastodon/presence.ts

```diff
@@ -0,0 +1,92 @@
+const presence = new Presence({
+		clientId: "1107321415243415574",
+	});
+
+
+// checks if user is logged from textarea
+function checkPositionAccountAvatar() {
+	if (document.querySelectorAll(".autosuggest-textarea__textarea").length < 1)
+		return 1;
+	return 2;
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/jlQv2if.png",
```

Comment:
Assets should be 512x512px in size.
Also, as your presence uses many assets, put them in an enum like so:
https://github.com/PreMiD/Presences/blob/3aa7e3fb8e54999012de1d1528f37bc907ae4213/websites/C/Crunchyroll/presence.ts#L52-L58